### ISSUE TITLE
port accumulateMethods and prefixedMethods to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5906]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5906|
|Type|enhancement|
|Created|2012-08-23 15:19:12Z|
|Branch|https://github.com/twisted/twisted/tree/some-reflect3-5906|

The unit tests for `SynchronousTestCase` depend on these two methods.

<details><summary>Searchable metadata</summary>

```
trac-id__5906 5906
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Python_3_3_Minimal Python 3.3 Minimal
branch__branches_some_reflect3_5906 branches/some-reflect3-5906
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1345735152000000 1345735152000000
changetime__1345824398000000 1345824398000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [35364]) Branching to 'some-reflect3-5906'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @itamarst|
|-|-|

I filed [#5909](https://github.com/twisted/twisted/issues/5909) for improving the docstrings here.  [Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/some-reflect3-5906)

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set owner to @exarkun|
|-|-|

Hooray for more test coverage.

  1. You should add a note to the test module referencing [#5885](https://github.com/twisted/twisted/issues/5885), for easy grepping of things that need fixing.
  1. I named the equivalent util module `_utilpy3`; either naming scheme is fine, we should just try to be consistent, so either rename yours or if you prefer this scheme I can rename mine.
  1. The X Y Z naming scheme is a tiny bit confusing; maybe "Base" and "Sub" and "method"?
  1. Needs news file.

Fix the above, then feel free to merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [35374]) Merge some-reflect3-5906

Author: exarkun
Reviewer: itamar
Fixes: [#5906](https://github.com/twisted/twisted/issues/5906)

Port a couple `twisted.python.reflect` methods to Python 3, in support of porting
a minimal subset of trial.  Also, add some unit tests for these previously
untested methods.
